installer/parse-install-yaml.py

```diff
@@ -31,19 +32,32 @@ def main():
 
         try:
             install_type = install_item["type"]
-            if install_type == "ros":
-                source_type = install_item["source"]["type"]
+            if install_type == "empty":
```

Comment:
We need to merge the empty target PR first, which has some unresolved issues.

Reply:
Aaah, that makes sense. I was actually wondering where the Empty statement went anyway.